### DISCUSSION TITLE
Modifying presets

### DISCUSSION THREAD

**Discussion by vaibhav0806** [fuma-nama/fumadocs#937] [Q&A]
[Here](https://fumadocs.vercel.app/docs/ui/theme#presets) as you have used an example preset `ocean`, I had some questions regarding presets:
- Are there more presets defined like this? Where and how can I see them?
- For an example preset like `ocean`, can I somehow see the code for this preset? 
- Finally, I want to make my custom preset for my docs, which is the best way to go about it? Like where to place the files and use it? Any best practices?

**Comment by fuma-nama** [comment]
1. See type definitions, it is the most accurate source
2. Check our source code in packages/ui/src, there’s a colors.ts file
3. Pass the preset object instead, see type definitions
